target/linux/realtek/image/rtl839x.mk

```diff
@@ -9,6 +9,22 @@ define Device/d-link_dgs-1210-52
 endef
 TARGET_DEVICES += d-link_dgs-1210-52
 
+define Device/hpe_1920-48g
+  $(Device/hpe_1920)
+  SOC := rtl8393
+  DEVICE_MODEL := 1920-48G (JG927A)
+  H3C_DEVICE_ID := 0x0001002a
+endef
+TARGET_DEVICES += hpe_1920-48g
+
+define Device/hpe_1920-48g-poe
+  $(Device/hpe_1920)
+  SOC := rtl8393
+  DEVICE_MODEL := 1920-48G-PoE (JG928A)
+  H3C_DEVICE_ID := 0x0001002b
```

Comment:
DEVICE_PACKAGES += realtek-poe should be added here for PoE support.

Reply:
Added those packages to the board.